projects/content-store/Makefile

```diff
@@ -1,2 +1,2 @@
-content-store: bundle-content-store router-api govuk-content-schemas
+content-store: bundle-content-store router-api govuk-content-schemas-app
```

Comment:
This won't work / doesn't need to change.

(Similarly for publishing-api.)

Reply:
Ah, I was getting test failures without this change 🤔

Reply:
Are you sure? I wonder if it was something unrelated.

Conversely, I get `make` failures _with_ this change.

Reply:
I've realised my mistake now, I needed to add the new dependency to docker-compose.yml, not the Makefile 🤦 Could you double-check my changes please?

Reply:
Hmm...nope 😄. I'll resolve this comment, but I've raised another...